mem/mem_bsd.go

```diff
@@ -0,0 +1,91 @@
+// +build freebsd openbsd
+
+package mem
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+const swapCommand = "/sbin/swapctl"
```

Comment:
Could you remove `/sbin/`?
```suggestion
const swapCommand = "swapctl"
```